### DISCUSSION TITLE
Clarification regarding PINT wildcard document type Identifier

### DISCUSSION THREAD

**Discussion by duke-nuke** [phax/phase4#287] [General]
This BIS supports two document type identifier schemes, busdox-docid-qns and peppol-doctype-wildcard,
what is the purpose/process of the peppol-doctype-wildcard document type identifier? 
do we need to define wildcard document type in SMP ? 

Regards

**Comment by phax** [comment]
This question is not so easy to answer at the moment :)
* PINT based document types are currently only relevant for Japan, AU-NZ, Malaysia and Singapore - for document types of other jurisdictions, PINT doesn't play a role atm
* The definition of the PINT document type identifier scheme is defined in the Policy for use of Identifiers - basically it should reduce the amount of SMP registrations, if there are multiple restricted (CIUS-like) document types registered in an SMP. However, we are currently in the middle of a transition between 2 different semantics, and the new one (Policy for use of Identifiers 4.3.0) must be usable from May 15th 2025. Until then, then Policy for use of Identifiers 4.2.0 is to be used. Please see also the migration guide from PFUOI 4.2 to 4.3: https://docs.peppol.eu/edelivery/changelog/2024-10/PINT%20Wildcard%20Migration%20Plan%201.0%202024.10.09.pdf for details on this

And yes, you need to register wildcard document types in your SMP if you want to receive them.
